fbw-a32nx/src/systems/instruments/src/EWD/PseudoFWC.ts

```diff
@@ -1220,6 +1220,7 @@ export class PseudoFWC {
         this.cpc2DiscreteWord.setFromSimVar('L:A32NX_PRESS_CPC_2_DISCRETE_WORD');
 
         const activeCpc = this.cpc1DiscreteWord.bitValueOr(11, false) ? this.cpc1DiscreteWord : this.cpc2DiscreteWord;
+        const activeCpcNumber = activeCpc === this.cpc1DiscreteWord ? 1 : 2;
```

Comment:
Why not do it this way?
```suggestion
        const activeCpcNumber = this.cpc1DiscreteWord.bitValueOr(11, false) ? 1 : 2;
        const activeCpc = activeCpcNumber  === 1 ? this.cpc1DiscreteWord : this.cpc2DiscreteWord;
```

Reply:
This also works 😛 Changed now.